node/src/Commands.ts

```diff
@@ -1974,6 +1974,61 @@ export function createXTrim(
     return createCommand(RequestType.XTrim, args);
 }
 
+export type StreamRangeBound =
```

Comment:
add StreamRangeBound to https://github.com/valkey-io/valkey-glide/blob/main/node/npm/glide/index.ts#L76-L249